src/clib/lib/res_util/res_env.cpp

```diff
@@ -66,6 +71,14 @@ char *res_env_alloc_PATH_executable(const char *executable) {
         int ipath = 0;
 
         for (auto path : path_list) {
+            try {
+                auto dir_iter = fs::directory_iterator(fs::path(path));
+            } catch (fs::filesystem_error &err) {
+                logger->warning(
+                    "Could not search PATH dir {} for executable {} - {}", path,
```

Comment:
Changed my mind here. We should probably silently ignore a directory in PATH we cannot open. That is what other tools do.

Reply:
fixed!